src/main/scala-sbt-1.0/com/typesafe/sbt/site/laika/LaikaSitePlugin.scala

```diff
@@ -0,0 +1,56 @@
+package com.typesafe.sbt.site.laika
+
+import java.io.File
+
+import sbt._
+import Keys._
+import com.typesafe.sbt.site.SitePlugin
+import com.typesafe.sbt.site.util.SiteHelpers
+import com.typesafe.sbt.site.SitePlugin.autoImport.siteSubdirName
+import com.typesafe.sbt.site.SitePlugin.autoImport.makeSite
+import laika.sbt.LaikaPlugin.autoImport.{Laika, laikaHTML, laikaSite }
+import laika.sbt.LaikaPlugin
+import Path.relativeTo
+
+object LaikaSitePlugin extends AutoPlugin {
```

Comment:
Unless there are considerable differences I suggest we keep the same plugin code. WDYT?

Reply:
The plugin code is virtually the same. Only some variable names have been refactored to reflect the changes in the laika plugin itself.

There is just this thing with the `target/laikaSite/site` that I cannot fix on my own. See comment above.